jekyll/_cci2/oss.md

```diff
@@ -83,11 +83,16 @@ If you are comfortable sharing secrets with anyone who forks your project and op
 
 ### Caching
 
-Caches are isolated based on GitHub Repo for PRs. CircleCI uses the GitHub repository-id of the originator of the fork PR to identify the cache.
-- PRs from the same fork repo will share a cache (this includes, as previously stated, that PRs in the master repo share a cache with master).
+Caches are isolated based on GitHub Repo for PRs. CircleCI uses the GitHub
+repository-id of the originator of the fork PR to identify the cache.
+- PRs from the same fork repo will share a cache (this includes, as previously
+  stated, that PRs in the master repo share a cache with master).
 - Two PRs in different Fork Repos will have different caches.
+- enabling the sharing of [environment variables]({{site.baseurl}}/2.0/env-vars)
+  with forked builds will enable cache sharing.
```

Comment:
This should probably say "cache sharing between origin and all forks" (or whatever our terminology is).

Reply:
updated.

Reply:
I don't know what our terminology is, but "origin" probably isn't it. It might be "origin repo" or "primary repo." If we introduce a new term, it may be unclear.

Reply:
Updated to `original repo`